app/assets/javascripts/test/snapshots/public/test-bundle/test/backend-snapshot-tests/annotations.e2e.js.md

```diff
@@ -4,6 +4,67 @@ The actual snapshot is saved in `annotations.e2e.js.snap`.
 
 Generated by [AVA](https://ava.li).
 
+## annotations-createExplorational
+
+    {
+      content: {
+        typ: 'skeleton',
+      },
+      dataSetName: 'confocal-multi_knossos',
+      dataStore: {
+        name: 'localhost',
+        typ: 'webknossos-store',
+        url: 'http://localhost:9000',
+      },
+      description: '',
+      isPublic: false,
+      name: '',
+      restrictions: {
+        allowAccess: true,
+        allowDownload: true,
+        allowFinish: true,
+        allowUpdate: true,
+      },
+      settings: {
+        allowedModes: [
+          'orthogonal',
+          'oblique',
+          'flight',
+        ],
+        branchPointsAllowed: true,
+        somaClickingAllowed: true,
+      },
+      state: 'Active',
+      stats: {},
+      tags: [
+        'confocal-multi_knossos',
+        'skeleton',
+      ],
+      task: null,
+      tracingTime: null,
```

Comment:
This probably needs a re-run once the PR for the sequential time logging is done.